orttraining/orttraining/core/session/training_session.h

```diff
@@ -27,6 +27,14 @@ class TrainingSession : public InferenceSession {
   typedef std::unordered_map<std::string /* Model weight name*/,
                              NameMLValMap /* 'Moment_1': OrtValue, 'Moment_2': OrtValue etc...*/>
                             OptimizerState;
+  
+  /**
+   * Partition information of each paritioned weight
+   */
+  struct PartitionInfo {
+    std::vector<int64_t> original_dim;
+    bool megatron_row_partition;
```

Comment:
can we make this an int and initialize by default to -1? That way, -1 indicates that megatron partitioning wasnt applied, 0 indicates row partition, 1 indicates column partition.

Reply:
Changed